recipes/pisces/meta.yaml

```diff
@@ -1,39 +1,34 @@
-{% set version = "5.2.10.49" %}
+{% set version = "5.3.0.0" %}
 
 package:
   name: pisces
   version: {{ version }}
 
 source:
-  url: https://github.com/Illumina/Pisces/releases/download/v{{ version }}/Pisces_{{ version }}.tar.gz
-  sha256: 03a9610e7b777e0a1a87e29f9334840f41732623b325250cbe4f5ad63338a093
+  # this fork does not require dotnet as a pre-req so friendly to linux
+  url: https://github.com/tamsen/Pisces/releases/download/v{{ version }}/pisces_all_{{ version }}.tar.gz
+  sha256: 4b9273b758271e8166bb75e1080896370f8cf3f1eaa23b0ffd71477a6004da15
 
 build:
   noarch: generic
-  number: 1
+  number: 0
   skip: False
```

Comment:
does it work on os x? release notes say that it is compiled for linux. may need to skip osx